STANDARD.md

```diff
@@ -448,22 +448,22 @@ short-hand notation (see: *Constant Record Components*).
                               with `constant record components`
     - example: reading example (with h5py) in Python:
 ```python
-def is_const_record(record_name, component_name) :
-    return ("value" in record_name["component_name"].attrs.keys())
+def is_const_record(record, component_name) :
+    return ("value" in record[component_name].attrs.keys())
 
-def get_component(record_name, component_name) :
-    if is_const_record(record_name, component_name) :
-        return record_name["component_name"].attrs["value"]
+def get_component(record, component_name) :
+    if is_const_record(record, component_name) :
+        return record[component_name].attrs["value"]
     else :
-        record_name["component_name"][:]
+        return record[component_name].value
 
 f = h5py.File('example.h5')
 species = f["<path_to_species_group>"]
 
 position_x_relative = get_component(species["position"], "x") \
-                    * species["position"].attrs["unitSI"]
+                    * species["position/x"].attrs["unitSI"]
```

Comment:
nope, it's not `scalar` in that sense. when we say `scalar` (1) or `vector` (>1) we mean the number of components of a record.

what you see in `dataset.attrs['value']` is a **constant** record component, which is an orthogonal concept that allows to save memory. On the other hand `dataset.value` is just `h5py` syntax to access the values inside a `h5py.Dataset` object.